includes/admin/class-wp-job-manager-taxonomy-meta.php

```diff
@@ -0,0 +1,156 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Handles taxonomy meta custom fields. Just used for job type.
+ *
+ * @package wp-job-manager
+ * @since 1.28.0
+ */
+class WP_Job_Manager_Taxonomy_Meta {
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var self
+	 * @since  1.28.0
+	 */
+	private static $_instance = null;
+
+	/**
+	 * Allows for accessing single instance of class. Class should only be constructed once per call.
+	 *
+	 * @since  1.28.0
+	 * @static
+	 * @return self Main instance.
+	 */
+	public static function instance() {
+		if ( is_null( self::$_instance ) ) {
+			self::$_instance = new self();
+		}
+		return self::$_instance;
+	}
+
+	/**
+	 * WP_Job_Manager_Taxonomy_Meta constructor.
+	 */
+	public function __construct() {
+		if ( wpjm_job_listing_employment_type_enabled() ) {
```

Comment:
I know we could just not include this file (and am happy with that) if disabled, but the future-purpose of this file is bigger than just just employment type meta.